lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1402,11 +1405,21 @@ def _create_node_runbook(
             azure_node_runbook.vhd = vhd
             azure_node_runbook.marketplace = None
             azure_node_runbook.shared_gallery = None
+            log.debug(
+                "there is no way to detect vhd generation, unless user provides it"
```

Comment:
The first sentence is not accurate, it may be the default value, not user provided. Change it like,

"current vhd generation is {azure_node_runbook.hyperv_generation}."

Reply:
updated